.changeset/great-colts-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
```

Comment:
```suggestion
'@backstage/cli': minor
```

This is a breaking change for users of the CLI.

Bit more discussion further down too, but I'm feeling it might make sense to ship this so let's move forward. What I want to make sure is that people have a reasonable quick fix for this when they do the version bump, and we should document it here. The fix should not be to have to hop into hundreds of files and add ESLint ignores or fix the imports straight away.

As far as I can tell, as long as you move the rules as I suggest further down, you can use the following to override the rule an allow MUI Links:

```js
module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
  restrictedImports: [
    { name: '@material-ui/core', importNames: [] },
    { name: '@material-ui/core/Link', importNames: [] },
  ],
});
```

I'm thinking we document that workaround for people that are bumping and running into this problem.